### PR TITLE
Update dragging framework wiki

### PR DESCRIPTION
**Pull request by PabstMirror**
From issue #4141

- remove notes about setMass
- add notes about weight limit variables

### CODE REVIEW

File: wiki/framework/dragging-framework.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ class CfgVehicles {
     <p>The following functions are NOT public and are likely to change in the future!</p>
 </div>
 
-You will **not** be able to carry / drag objects that are too heavy, the mass is also affected by what is inside the object. To bypass this empty the object and/or use `setMass`.
+You will **not** be able to carry / drag objects that are too heavy, the mass is also affected by what is inside the object. To bypass this empty the object. You can change the weight limits by setting `ACE_maxWeightDrag` (default 800) and `ACE_maxWeightCarry` (default 600).
```
Comment (bux578) [inline wiki/framework/dragging-framework.md]: > by setting `ACE_maxWeightDrag` (default 800) and `ACE_maxWeightCarry` (default 600).

Is this a setting? If so, could you edit it to something like `by changing the following ACE3 settings `
(Do we have a module for this? http://ace3mod.com/wiki/missionmaker/modules.html)

Comment (PabstMirror) [inline wiki/framework/dragging-framework.md]: It's just a raw variable in missionNamespace

Comment (bux578) [inline wiki/framework/dragging-framework.md]: ah, ok. Maybe an example?

Comment (esteldunedain) [inline wiki/framework/dragging-framework.md]: > It's just a raw variable in missionNamespace

We should make it a setting when the new system is adopted

**Pull request merged**